src/__tests__/plugins-test.js

```diff
@@ -171,27 +171,6 @@ test("module providing an array of plugins", t => {
   t.plan(planned)
 })
 
-test("deprecation warning for slashless plugin rule names", t => {
```

Comment:
Should this test be replaced with a test to check for a config error? Or should the file `plugin-slashless-warn-about-foo.js` be removed?